modules/command/src/main/java/org/apache/fluo/command/FluoList.java

```diff
@@ -91,7 +91,8 @@ private void listApp(FluoConfiguration config, String path) {
         state = "RUNNING";
       }
       int numWorkers = admin.numWorkers();
-      System.out.format("%-15s %-11s %4d\n", path, state, numWorkers);
+      int numOracles = admin.numOracles();
+      System.out.format("%-15s %-11s %4d %13d\n", path, state, numWorkers, numOracles);
```

Comment:
This is a really neat change.  Would you happen to have any example output from running?  If not don't worry about it.

Reply:
I've attached some screenshots so you can see how the spacing looks.

As a note, if you call `fluo list` too soon after starting or stopping an oracle then the oracle count may not have been updated yet (you can see some examples of this in my screenshots). I see the same behavior with workers so I think it should be fine. 
![Screenshot from 2020-03-02 20-33-10](https://user-images.githubusercontent.com/10800303/75734230-61f54c80-5cc5-11ea-8ca7-d4b9c4015436.png)
![Screenshot from 2020-03-02 20-33-59](https://user-images.githubusercontent.com/10800303/75734233-628de300-5cc5-11ea-85db-6e930372faa8.png)